control/control.SLES.xml

```diff
@@ -309,21 +309,27 @@ Please visit us at http://www.suse.com/.
         <service_sfcb><label>CIM Server</label></service_sfcb>
 
         <normal_role>
+          <!-- translators: a label for a system role -->
```

Comment:
Um, I thought the convention is to use `TRANSLATORS:` (capitalized) comment. https://en.wikipedia.org/wiki/Gettext